### DISCUSSION TITLE
How to remove this line on top of file?

### DISCUSSION THREAD

**Discussion by krutoo** [gitkraken/vscode-gitlens#1848] [Q&A] [answered]
Hi, how can i remove this line on top of file content?

<img width="440" alt="example of git info on top of file in VSCode" src="https://user-images.githubusercontent.com/13140130/156105596-1421d12a-a397-4a7f-9842-8d10e76acec6.png">

I mean text "You, 2 minutes ago | 12 authors (dima petrov and others)"

**Comment by eamodio** [comment, marked as answer]
From the Command Palette, enter `GitLens: Open Settings` and then jump/scroll to the `Git CodeLens` section:

![image](https://user-images.githubusercontent.com/641685/156113952-72f7a398-c325-404f-a046-6df1ecf07fca.png)

From there, you can disable the CodeLens altogether, or if you just want to remove it from the top of the file, you can uncheck the `File scope` checkbox. Playing with those settings will be reflected in the preview image.

Hope that helps!

**Reply by krutoo** [reply]
thank you so much
